@xen-orchestra/lite/src/App/TreeView.tsx

```diff
@@ -33,10 +35,10 @@ const getHostPowerState = (host: Host) => {
   return $metrics ? ($metrics.live ? 'Running' : 'Halted') : 'Unknown'
 }
 
-const getIconColor = (obj: Object) => {
+const getIconColor = (obj: Host | Vm) => {
   let powerState = obj.power_state
   if (obj.$type === 'host') {
```

Comment:
I would prefer using the same check instead of one on $type and one with a cast 
```suggestion
  if (obj as Host) {
```